static/app/views/explore/logs/useSaveAsItems.spec.tsx

```diff
@@ -257,7 +257,7 @@ describe('useSaveAsItems', () => {
             end: '2024-01-01T01:00:00.000Z',
             range: '1h',
             environment: ['production'],
-            interval: '5m',
```

Comment:
Given that logs were already setting `ChartIntervalUnspecifiedStrategy.USE_SMALLEST` I'm confused as to why I needed to update this test.

Would like to look a little closer before landing.